dll/win32/comdlg32/lang/cdlg_Pt.rc

```diff
@@ -301,34 +301,34 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Procurar"
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT "&Procurar:", -1, 4, 8, 52, 8 //FIXME: accelerator collision &P
+    LTEXT "&Procurar:", -1, 4, 8, 52, 8
     EDITTEXT edt1, 57, 7, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
     CHECKBOX "Palavra &inteira", chx1, 4, 26, 140, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
     CHECKBOX "&Maiúsculas/minúsculas", chx2, 4, 42, 140, 12, BS_AUTOCHECKBOX | WS_TABSTOP
     GROUPBOX "Direcção", grp1, 147, 21, 58, 38
-    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP//FIXME: accelerator collision &A
+    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP
     AUTORADIOBUTTON "A&baixo", rad2, 151, 44, 48, 12, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    DEFPUSHBUTTON "&Procurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON //FIXME: accelerator collision &P
+    DEFPUSHBUTTON "Pr&ocurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
     PUSHBUTTON "Cancelar", IDCANCEL, 212, 24, 60, 14, WS_GROUP | WS_TABSTOP
-    PUSHBUTTON "&Ajuda", pshHelp, 212, 42, 60, 14, WS_GROUP | WS_TABSTOP //FIXME: accelerator collision &A
+    PUSHBUTTON "A&juda", pshHelp, 212, 42, 60, 14, WS_GROUP | WS_TABSTOP
 }
 
 REPLACEDLGORD DIALOG 36, 24, 276, 94
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Substituir"
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT "&Procurar:", -1, 4, 8, 52, 8 //FIXME: accelerator collision &P
+    LTEXT "&Procurar:", -1, 4, 8, 52, 8
     EDITTEXT edt1, 57, 7, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
     LTEXT "Substituir po&r:", -1, 4, 26, 52, 8
     EDITTEXT edt2, 57, 24, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
     CHECKBOX "Palavra &inteira", chx1, 5, 46, 120, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
     CHECKBOX "&Maiúsculas/minúsculas", chx2, 5, 62, 120, 12, BS_AUTOCHECKBOX | WS_TABSTOP
-    DEFPUSHBUTTON "&Procurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON //FIXME: accelerator collision &P
+    DEFPUSHBUTTON "Pr&ocurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
```

Comment:
```suggestion
    DEFPUSHBUTTON "Pr&ocurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP
```

Reply:
Yes, but I guess it is better to use a fresh PR for that, that would cover all languages, and not just pt-PT.

Reply:
Ok i can doit. Thanks.